packages/gatsby-theme-store/src/sdk/search/useSearchFiltersFromPageContext.ts

```diff
@@ -58,7 +58,11 @@ export const useSearchFiltersFromPageContext = (
 
   return useMemo(() => {
     const params = new URLSearchParams(search)
-    let { query, map } = pageContext
+    // Quick fix for some questions regarding memo dependencies
+    /* eslint-disable-next-line prefer-destructuring */
+    let query = pageContext.query
+    /* eslint-disable-next-line prefer-destructuring */
+    let map = pageContext.map
```

Comment:
You can fix this eslint ignore with:

```tsx
const { query, staticPath, query, map, orderBy, selectedFacets } = pageContext

useMemo(() => {


}, [search, staticPath, query, map, orderBy, selectedFacets, pathname])
```

Reply:
@klzns  Good idea, but I'm just gonna merge for now and do this in a future PR